content/docs/introducing-jsx.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: introducing-jsx
-title: Introducing JSX
+title: पेश है JSX
 permalink: docs/introducing-jsx.html
 prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+इस परिवर्तनशील घोषणा पर विचार करें::
```

Comment:
परिवर्तनशील घोषणा -> too complex

Reply:
how abot `वेरिएबल घोषणा` or `वेरिएबल डेक्लेरेशन` or should we not translate variable
`variable डेक्लेरेशन`

which one sounds better? Any suggestions?

Reply:
either one of `वेरिएबल घोषणा` or `वेरिएबल डेक्लेरेशन` looks good